### PR TITLE
KCP-77 - UI model updates

### PR DESCRIPTION
**Pull request by phuhgh**
https://wintoncentre.atlassian.net/browse/KCP-77

This PR introduces the model changes to make use of competing mortality. There are outstanding features / bugs in the UI to be addressed in other issues.

### CODE REVIEW

File: src/kcp/results.cljs [hunk 1]
```diff
@@ -37,74 +95,77 @@
...
         sum-betas (map #(fac/sum-beta-xs context %) beta-keys)
-        cox? (model/use-cox-adjusted? tool)
-        F (if cox?
-            (model/cox-adjusted s0 sum-betas)
-            (model/cox-only s0 sum-betas))
-
-        sample-days (map
-                      utils/year->day
-                      (if s0 (range (inc (utils/day->year (first (last s0))))) []))
+        F (normalize-vectors (merge-vectors (model/cox-only s0 sum-betas) mortality-data))
```
Comment (phuhgh) [inline src/kcp/results.cljs:160]: @hanharrison92  new normalization code

File: test/kcp/vis2_test.cljs [hunk 2]
```diff
@@ -81,40 +62,15 @@
...
-  (->> (gen/sample (generate-fs {:length 3}) 100)
-       (map vis/int-fs-series)
-       (map :int-fs)
-       (map #(apply + %)))
-  
-  (map vis/int-fs-series
-       (gen/sample (generate-fs {:length 3}) 100))
 
-  (gen/vector gen/small-integer)
+(deftest int-fs-series
```
Comment (phuhgh) [inline test/kcp/vis2_test.cljs:66]: @hanharrison92 these 3

File: src/kcp/results.cljs (hunk 1 above)
Comment (phuhgh) [reply src/kcp/results.cljs]: Alternately, if you can point me at the discrepancy I'll debug it

Comment (hanharrison92) [reply src/kcp/results.cljs]: Sorry, taken me a while to double check everything on our end. Will email some test cases.

Comment (hanharrison92) [reply src/kcp/results.cljs]: Double checked, no discrepancies found in the implementation of the updated algorthim.

**Pull request closed without merging**